docs/content/configuration/index.md

```diff
@@ -1251,7 +1251,11 @@ These Historical configurations can be defined in the `historical/runtime.proper
 |`druid.segmentCache.infoDir`|Historical nodes keep track of the segments they are serving so that when the process is restarted they can reload the same segments without waiting for the Coordinator to reassign. This path defines where this metadata is kept. Directory will be created if needed.|${first_location}/info_dir|
 |`druid.segmentCache.announceIntervalMillis`|How frequently to announce segments while segments are loading from cache. Set this value to zero to wait for all segments to be loaded before announcing.|5000 (5 seconds)|
 |`druid.segmentCache.numLoadingThreads`|How many segments to drop or load concurrently from from deep storage.|10|
-|`druid.segmentCache.numBootstrapThreads`|How many segments to load concurrently from local storage at startup.|Same as numLoadingThreads|
+|`druid.coordinator.loadqueuepeon.curator.numCreateThreads`|Number of threads creating zk nodes corresponding to segments that need to be loaded or dropped.|10|
```

Comment:
Why 10? Why not dynamic, like for `numLoadingThreads`? How and why would one possibly want to change this configuration, in response to what conditions? Documentation should answer these questions.

Same questions about other newly added configuration parameters.

Reply:
10 is the default value.

Reply:
Yeah, but what is the logic behind it? In the very same PR, you have changed the default number of threads for other thread pool from 10 to `N_CPU / 2`, why not follow the same logic here?